backend/LexBoxApi/Models/Project/ProjectMemberInputs.cs

```diff
@@ -5,4 +5,8 @@ namespace LexBoxApi.Models.Project;
 
 public record AddProjectMemberInput(Guid ProjectId, [property: EmailAddress] string UserEmail, ProjectRole Role);
 
+public record BulkAddProjectMembersInput(Guid ProjectId, string[] Usernames, ProjectRole Role, string PasswordHash);
+
+public record BulkAddProjectMembersResult(LexCore.Entities.Project project, int CreatedCount, List<string> UsernameConflicts);
```

Comment:
`project` should be `Project` to follow standard record naming convention

Reply:
Moot now that we've removed the project property in 86c9b78379c26ea19eab2d2b63f07d04f020abfb.